test/test.js

```diff
@@ -3,17 +3,35 @@ import 'babel-polyfill';
 
 import assert from 'assert';
 import { exec } from 'mz/child_process';
-import { exists, readFile, writeFile } from 'mz/fs';
+import { exists, readFile, writeFile, mkdtemp, mkdir } from 'mz/fs';
```

Comment:
Actually, looks like mkdtemp was introduced in node 5, and I'd like to support node 4, so probably best to stick with the old random number approach unless there's a problem with it.